src/Category/Topos.ard

```diff
@@ -7,43 +7,63 @@
 \import Paths.Meta
 \open CartesianPrecat
 
-\class ElementaryTopos \extends FinCompletePrecat, Cat
+\class Topos \extends FinCompletePrecat, Cat
```

Comment:
What about Grothendieck topos?

Reply:
I have no idea. I think we should ask valis. Why do you suggest E topos to be those who have an NNO? Is this a convention?

P.S. I hardly know category theory, and this might be a dumb question, don't overthink